Chaincase.UI/ViewModels/NewPasswordViewModel.cs

```diff
@@ -32,6 +32,7 @@ public void SetPassword(string password)
 
 			Global.UiConfig.HasSeed = true;
 			Global.UiConfig.ToFile();
+			Global.SetDefaultWallet();
```

Comment:
y tho

in index, no?

Reply:
When you set the password, doesn't this mean a new wallet is saved? Which would mean you need to reload the loaded wallet if there is one?

Reply:
whenever the index view is loaded a check happens hand SetDefaultWallet is called I believe. I don't think this causes any ill effects, though

Reply:
The IndexViewModel is a singleton, so if you change wallet after it was constructed, it would not load the new one

Reply:
D'oh! of course. Thanks for the catch